client/grpc_query_test.go

```diff
@@ -1,3 +1,5 @@
+// +build norace
```

Comment:
How do these cause data races? Should this not be fixed instead of introducing a build flag?

Reply:
The new test suites cause data races, see https://github.com/cosmos/cosmos-sdk/pull/7300

Reply:
could an issue be opened to fix this?

Reply:
I promise I'll open one :)